test/PrintAsObjC/availability.swift

```diff
@@ -148,71 +200,169 @@
     @available(tvOSApplicationExtension, unavailable)
     @available(watchOSApplicationExtension, unavailable)
     @objc func extensionUnavailable() {}
+  
+    @available(*, deprecated, renamed: "overloadMethod1(first:second:)")
+    @objc func deprecatedMethodRenamedToOverloadMethod1(first: Int, second: Int) {}
+  
+    @objc(overloadMethodWithFirst:second:) func overloadMethod1(first: Int, second: Int) {}
+    @objc func overloadMethod2(first: Double, second: Double) {}
```

Comment:
Wait, these aren't overloads if you give them different base names.

Reply:
OH what a mistake!!!